components/automate-cli/Makefile

```diff
@@ -59,4 +59,4 @@ linux darwin windows:
 	@echo Done: ${CROSS_TARGET_DIR}/${BUILD_TARGET}
 
 unit:
-	@go test --count 1 $(shell go list ./cmd/... ./pkg/...)
+	@go test --count 1 ./...
```

Comment:
Based on the pattern "go test with go list should change to this", also consider:
api/Makefile